src/components/toast.js

```diff
@@ -0,0 +1,13 @@
+import { toast } from "react-toastify";
+
```

Comment:
Can we set the position to top-right

Reply:
When the position is top-right, it's hiding the logout button. Is that okay?

Reply:
Display in middle is not suitable I think. I think it is no matter coz toast will disappear after few seconds. If it is a issue we can set position to bottom-right, right ?

Reply:
Okay, then will set it to top-right itself, because as you said it'll disappear after a few seconds.